.golangci.yaml

```diff
@@ -34,3 +34,7 @@ linters:
   - usestdlibvars
   - wastedassign
   - whitespace
+
+issues:
+  exclude:
+    - "Error return value of `os\\.MkdirAll` is not checked" # commonly unchecked error
```

Comment:
I guess that's mostly related to creating dir structure during startup, like certificates and stuff. Sounds like something worth catching